scio-google-cloud-platform/src/main/java/com/spotify/scio/bigtable/BigtableUtil.java

```diff
@@ -85,8 +89,15 @@ public static void updateNumberOfBigtableNodes(
       final ListClustersResponse clustersResponse =
           bigtableInstanceClient.listCluster(clustersRequest);
 
+      final List<Cluster> clusters = clustersResponse
+          .getClustersList()
+          .stream()
+          // if no clusters specified then apply to all clusters in the instance.
+          .filter(c -> clusterNames.isEmpty() || clusterNames.contains(shorterName(c.getName())))
```

Comment:
Since we are passing a set of cluster names the expectation should be either:
* update if they exist
* or none if otherwise. We should also return an error i.e `IllegalArgumentExcpetion` if that's the case.

It would be nice to keep the previous method as well.

Reply:
Refactored the code; made clusterNames an optional type and throw exceptions if it is an empty set.